flink-connectors/flink-connector-hbase-base/src/main/java/org/apache/flink/connector/hbase/sink/HBaseSinkFunction.java

```diff
@@ -76,6 +79,7 @@
     private transient ScheduledExecutorService executor;
     private transient ScheduledFuture scheduledFuture;
     private transient AtomicLong numPendingRequests;
+    private static Map<byte[], Mutation> mutationMap = new HashMap<>();
```

Comment:
Why is this static? This means subtasks on the same JVM would all have access to, and try to flush the same data.

Reply:
Thank you for your review. I agree with you. I try to make it be the global queue to reduce the rowkey before. However, in this case, In fact, there is no need to declare it as static.

Reply:
I will fix it later.